apps/asap-server/src/queries/teams.queries.ts

```diff
@@ -119,7 +119,7 @@ export const FETCH_TEAMS = gql`
     $top: Int
     $skip: Int
     $filter: String
-    $withResearchOutputs: Boolean = true
```

Comment:
do we ever fetch them WITH reasearch outputs?

Reply:
ok so by NOT removing the reasearch outputs and only changing the default value for the Directive you effectively didn't alter the types for the response of the query - this is the downside of these generated queries which, if such directive is ever used, ALWAYS include the conditional resource. 

If there is a valid reason for keeping the directive and the flag here then leave it but otherwise i would remove this part entirely:
https://github.com/yldio/asap-hub/blob/b369fb43363491a9132e1098133ec68f1587a8f2/apps/asap-server/src/queries/teams.queries.ts#L13-L15